### ISSUE TITLE
Remove ONE OF THE duplicated PLUGINS StatesDumper and StorageDumper

### ISSUE THREAD

**Issue by vncoelho** [neo-project/neo#3271]
These two plugins were created with very similar ideas.
They can be merged into one with parameters or functions to guide its functionality.

It will also improve maintenance and organization.

**Comment by vncoelho** [comment]
@superboyiii, can you confirm that? I believe that one of them was created by you, and the other by Igor and I.

**Comment by roman-khimov** [comment]
Makes sense. My only concern is compatibility here, we (and I suspect we're not alone) have some tooling that uses `StorageDumper` output.

**Comment by Jim8y** [comment]
Good idea but plugins are already there, can merge into one, but keep the original.

**Comment by vncoelho** [comment]
> Makes sense. My only concern is compatibility here, we (and I suspect we're not alone) have some tooling that uses `StorageDumper` output.

This is the one we originally created. Glad to know you use it.

I am sure keep compatibility, maybe just the name of the plugin will change.

**Comment by cschuchardt88** [comment]
I can make a new one If needed, with the two combined. I would be rewriting it.

**Comment by vncoelho** [comment]
let me do this with time, there is no rush in this task.

**Comment by vncoelho** [comment]
> I can make a new one If needed, with the two combined. I would be rewriting it. Also would be more features and Unit tests, If needed.

but I appreciate your initiative, @cschuchardt88

**Comment by Jim8y** [comment]
> I can make a new one If needed, with the two combined. I would be rewriting it. Also would be more features and Unit tests, If needed.

vitor started, should let vitor do it.

**Comment by vncoelho** [comment]
I did not yet started and it may take time...aheuahuea

**Comment by vncoelho** [comment]
@superboyiii @roman-khimov 

I checked here with careful.
Both plugins are the same. ahueaheuaea
There are just some minor differences in the saving procedure of each of them.

Have you been using one of these plugins?
If so, which of them?

**Comment by vncoelho** [comment]
I updated the title from unify to remove duplicated.
There is nothing to UNIFY from these two plugins.

I checked the history on `neo-modules` and also verified code here.
I do not know why, in the past, @Ashuaidehao and @superboyiii pushed this new module instead of refactoring `StateDumper`.
Since then, we have these two DUPLICATED code.

As @roman-khimov said he was using `StorageDumper` and I also verified its code is more compact than my previous `StateDumper` plugin.
Thus, I kept `StorageDumper`
